src/component/registry/bpmn-elements-registry.ts

```diff
@@ -61,7 +62,10 @@ export class BpmnElementsRegistry {
     private cssRegistry: CssRegistry,
     private graphCellUpdater: GraphCellUpdater,
   ) {
-    this.bpmnModelRegistry.registerOnLoadCallback(this.cssRegistry.clear);
+    this.bpmnModelRegistry.registerOnLoadCallback(() => {
+      this.cssRegistry.clear();
+      this.graphCellUpdater.styleManager.clear();
```

Comment:
**suggestion:** this expose too much internal of the cell updater. I suggest to have a clear function in CellUpdater for simplification

Reply:
Good catch ^^